packages/mui-material/src/styles/CssVarsProvider.test.js

```diff
@@ -140,7 +140,6 @@ describe('[Material UI] CssVarsProvider', () => {
           primary: 'var(--mui-palette-text-primary)',
           secondary: 'var(--mui-palette-text-secondary)',
           disabled: 'var(--mui-palette-text-disabled)',
-          icon: 'var(--mui-palette-text-icon)',
```

Comment:
Need to remove this since `extendTheme()` produces only light by default and `icon` exists in dark palette only